bundle/src/main/resources/angularjs-partials/propertyInputTextarea.html

```diff
@@ -3,8 +3,10 @@
     <textarea is="coral-textarea"
               ng-model="property.value" ng-trim="false"
               ng-required="{{property.metadata.properties.required && !property.overridden && !property.inherited}}"
+              caconfig-validation="{{validation}}"
               name="textArea"></textarea>
     <div class="coral-Form-errorlabel" bindonce="i18n('validation.required')" ng-show="inputTextAreaForm.textArea.$error.required" bo-text="i18n('validation.required')"></div>
+    <div class="coral-Form-errorlabel" ng-if="validationMessage" ng-show="inputTextAreaForm.textArea.$error.caconfigValidation || inputTextAreaForm.textArea.$pending.caconfigValidation">{{validationMessage}}</div>
```

Comment:
i've tested this with https://github.com/wcm-io/io.wcm.caconfig.sample-app/commit/fc20ceffdaa7742b6d2c4b3be20f4f94429cd7a7, but i'm not seeing the validation message below the text area. can you cross-check?